### PR TITLE
Use null coalescing operator instead of isset

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by mvorisek** [comment]
There is a small bug - see https://github.com/atk4/data/runs/570500993

How to solve it?

File: src/Persistence.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public static function connect($dsn, $user = null, $password = null, $args = [])
         // Process DSN string
         $dsn = \atk4\dsql\Connection::normalizeDSN($dsn, $user, $password);
 
-        $driver = isset($args['driver']) ? strtolower($args['driver']) : $dsn['driver'];
+        $driver = strtolower($args['driver'] ?? $dsn['driver']);
```
Comment (DarkSide666) [inline src/Persistence.php:43]: This is slight actual change, but I think that's ok.

Comment (mvorisek) [reply src/Persistence.php:43]: Step by step improvement ;-)

**Comment by mvorisek** [comment]
> 
> 
> Overall looks good. I will check why value type is cast in this case. I guess `??` does that somehow.

No, see the 2nd reverted commit - the issue is that the return type is now forced to string (by function header). Probably the export function should not use toString for numerical fields.

**Comment by DarkSide666** [comment]
> No, see the 2nd reverted commit - the issue is that the return type is now forced to string (by function header). Probably the export function should not use toString for numerical fields.

Oh I see. Well... I think `toString` should return string always, so maybe we should change tests anxd cast everything to string. I think it will not affect anything important because it still will be `var_dump("5"+3);` int 8

**Comment by mvorisek** [comment]
@DarkSide666 Done, same assumption on my side, but the test data come from `export` method and I think this method should preserve the integer type.

Any idea how to fix this properly, i.e. without changing the test data?

**Comment by DarkSide666** [comment]
No idea, but I think it's OK this way.
If we want typecasted export, then there is 3rd parameter of export method. Otherwise it's not typecasted - so doing toString.
These `vat_rate_id` fields are not actually id fields. They are not defined as hasOne fields I think.

**Pull request merged**